.gitignore

```diff
@@ -46,3 +46,5 @@ browse.VC.db
 CMakeLists.txt.user
 /CMakeSettings.json
 /.vs
+/.cproject
+/.project
```

Comment:
Not really part of this PR, but probably fine to just leave it in here.

Reply:
This is part of the original PR. I agree however. On the other hand, if we don't include them now, they'll never be included, because such a tiny PR would be very far at the end of the PRs to be reviewed. :)

Reply:
What IDE uses these?

Reply:
Eclipse CDT and maybe the latter by IntelliJ IDEA (Never used either of them, but google helped me out) :-)

Reply:
Yes, I had used Eclipse CDT.